ballerina/tests/generated_client.bal

```diff
@@ -197,6 +210,36 @@ client class RainierClient {
         return 'object;
     };
 
+    isolated resource function get orderItems() returns stream<OrderItem, error?> {
+        stream<record{}, sql:Error?>|Error result = self.persistClients.get(ORDER_ITEM).runReadQuery(OrderItem);
+        if result is Error {
+            return new stream<OrderItem, Error?>(new OrderItemStream((), result));
+        } else {
+            return new stream<OrderItem, Error?>(new OrderItemStream(result));
+        }
+    };
+
+    isolated resource function get orderItems/[string orderId]/[string itemId]() returns OrderItem|error {
+        return (check self.persistClients.get(ORDER_ITEM).runReadByKeyQuery(OrderItem, {orderId: orderId, itemId: itemId})).cloneWithType(OrderItem);
+    };
+
+    isolated resource function post orderItems(OrderItemInsert[] data) returns [string, string][]|error {
+        _ = check self.persistClients.get(ORDER_ITEM).runBatchInsertQuery(data);
+        return from OrderItemInsert inserted in data
+               select [inserted.orderId, inserted.itemId];
+    };
+
+    isolated resource function put orderItems/[string orderId]/[string itemId](OrderItemUpdate data) returns OrderItem|error {
```

Comment:
```suggestion
    isolated resource function put orderItems/[string orderId]/[string itemId](OrderItemUpdate data) returns OrderItem|persist:Error {
```
It should be changed in all resources.

Reply:
Fixed in [002ef6f](https://github.com/ballerina-platform/module-ballerina-persist/pull/105/commits/002ef6fc58d62842bfd718458b5063f38333ea5c)